definitions/pipelines/rnaseq_star_fusion.cwl

```diff
@@ -89,9 +89,19 @@ outputs:
     fusion_evidence:
         type: File
         outputSource: kallisto/fusion_evidence
+<<<<<<< HEAD
```

Comment:
Seems like you've got merge conflicts still in here

Reply:
ugg conflict markers..